src/tests/JIT/Regression/JitBlue/Runtime_85630/Runtime_85630.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Optimize>True</Optimize>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildProjectName).cs" />
```

Comment:
did you try manually reducing the test? Honestly, this is too big test case to include.

Reply:
I can try to reduce it down.